examples/with-next/pages/editor/social/content.tsx

```diff
@@ -2,7 +2,11 @@ import { ExampleSocialEditor, SOCIAL_SHOWCASE_CONTENT } from '@remirror/showcase
 import React, { FC } from 'react';
 
 const SocialEditorWithContent: FC = () => (
-  <ExampleSocialEditor initialContent={SOCIAL_SHOWCASE_CONTENT} suppressHydrationWarning={true} />
+  <ExampleSocialEditor
+    initialContent={SOCIAL_SHOWCASE_CONTENT}
+    suppressHydrationWarning={true}
+    characterLimit={280}
+  />
```

Comment:
![Screenshot_20200109_124448](https://user-images.githubusercontent.com/129910/72068856-d91dee00-32dd-11ea-8e0b-0f0c06d8e4f9.png)

This used to have 2 characters remaining, now it has 142.